lib/xboxrt/vcruntime/excpt.cpp

```diff
@@ -0,0 +1,201 @@
+#include <excpt.h>
+#include <excpt_cxx.hpp>
+#include <eh.h>
+#include <xboxkrnl/xboxkrnl.h>
+
+#define MAGIC_VC  0x19930520 // up to VC6
+#define MAGIC_VC7 0x19930521 // VC7.x(2002-2003)
+#define MAGIC_VC8 0x19930522 // VC8 (2005)
```

Comment:
Do these exist for more modern VC versions, too? what happened afterwards? Why is this tag necessary in the first place?

Reply:
Afaik the C++ exception ABI stayed the same after VC8, so no new magic values seemed to appear.